project.clj

```diff
@@ -133,6 +133,7 @@
    [ring/ring-core "1.8.0"]
    [ring/ring-jetty-adapter "1.8.0"]                                  ; Ring adapter using Jetty webserver (used to run a Ring server for unit tests)
    [ring/ring-json "0.4.0"]                                           ; Ring middleware for reading/writing JSON automatically
+   [ring/ring-ssl "0.3.0"]                                            ; Ring middleware for redirecting HTTP to HTTPS
```

Comment:
I didn't know this existed. Does it do what we want?

Reply:
not everything - it can't do the conditions we want, but it's still useful